tests/integration/conftest.py

```diff
@@ -64,4 +74,57 @@ def time_series(request, cognite_client):
     create_subscription_in_cdf(timeseries_set, sub_name, cognite_client)
     yield timeseries_set
     remove_time_series_data(timeseries_set, cognite_client)
-    cognite_client.time_series.subscriptions.delete(sub_name)
\ No newline at end of file
+    cognite_client.time_series.subscriptions.delete(sub_name)
+
+@pytest.fixture(scope="session")
+def test_space(cognite_client: CogniteClient):
+    space = cognite_client.data_modeling.spaces.retrieve(SPACE_ID)
+    if space is not None:
+        yield space
+    else:
+        new_space = SpaceApply(SPACE_ID, name="Integration Test Space", description="The space used for integration tests.")
+        yield cognite_client.data_modeling.spaces.apply(new_space)
+    cognite_client.data_modeling.spaces.delete(spaces=[SPACE_ID])
```

Comment:
Rather than having 2 yields it might be clearer to do this:
```
    space = cognite_client.data_modeling.spaces.retrieve(SPACE_ID)
    if space is None:
        new_space = SpaceApply(SPACE_ID, name="Integration Test Space", description="The space used for integration tests.")
        space = cognite_client.data_modeling.spaces.apply(new_space)
    yield space
```

Reply:
Done